stack/core/src/main/java/org/apache/usergrid/persistence/cassandra/CassandraService.java

```diff
@@ -158,7 +164,7 @@ public void init() throws Exception {
         accessMap.put( "username", properties.getProperty( "cassandra.username" ) );
         accessMap.put( "password", properties.getProperty( "cassandra.password" ) );
         systemKeyspace =
-                HFactory.createKeyspace( SYSTEM_KEYSPACE, cluster, consistencyLevelPolicy, ON_FAIL_TRY_ALL_AVAILABLE,
+                HFactory.createKeyspace( getApplicationKeyspace() , cluster, consistencyLevelPolicy, ON_FAIL_TRY_ALL_AVAILABLE,
```

Comment:
Can we remove hector calls to create keyspace and our system functions?  The CP modules are more intelligent in their creation of the keyspaces.  If we can remove it here, lets do that.  Note that hector can still create CFs, but the RF settings on the keyspaces seem to not work well in practice.